src/main/java/de/cas/mse/exercise/SumDigPowerCalculator.java

```diff
@@ -0,0 +1,41 @@
+package de.cas.mse.exercise;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SumDigPowerCalculator {
+
+	public List<Long> calculate(long lowerBoundary, long upperBoundary) {
```

Comment:
`calculate` sounds a bit like a single value being calculated, that might be misleading. The method collects or filters (and prints as a side effect).